src/Psl/Xml/Exception/XmlException.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\Xml\Exception;
+
+use Exception;
+use Psl\Exception\RuntimeException;
+use Psl\Xml\Error\XmlErrorCollection;
+
+class XmlException extends RuntimeException
```

Comment:
we should add `Psl\Xml\Exception\ExceptionInterface` here ( as we do with every other component )